web/gds-user-ui/src/components/Contacts/index.tsx

```diff
@@ -1,18 +1,38 @@
-import { Heading, HStack, Stack, Text } from '@chakra-ui/react';
+import { Heading, HStack, Stack, Text, chakra } from '@chakra-ui/react';
 import { t } from '@lingui/macro';
 import { Trans } from '@lingui/react';
 import { getCurrentStep, getSteps } from 'application/store/selectors/stepper';
-import ContactForm from 'components/ContactForm';
+import ContactForm from 'components/Contacts/ContactForm';
 import { SectionStatus } from 'components/SectionStatus';
 import FormLayout from 'layouts/FormLayout';
 import { useSelector } from 'react-redux';
 import { getStepStatus } from 'utils/utils';
+import { contactsValidationSchema } from 'modules/dashboard/certificate/lib/contactsValidationSchema';
+import StepButtons from 'components/StepsButtons';
+// import { ContactsForm } from './ContactsForm'; // group all contact forms in one component
```

Comment:
Is there a separate story for this?

Reply:
@daniellemaxwell  No should be done in Refactor Contact Form story. it should be the same implementation as I did for the new LegalForm component. Let me know if you need more clarification. 

PS: *ContactsForm should be implemented the same way like LegalForm component*